packages/components/src/components/autocomplete/autocomplete.jsx

```diff
@@ -5,6 +5,7 @@ import { getPosition } from '@deriv/shared';
 import Icon from '../icon';
 import Input from '../input';
 import DropdownList from '../dropdown-list';
+import { useBlockScroll } from '../../hooks/use-blockscroll';
```

Comment:
Shouldn't this be in `shared` package?

Reply:
No, because we have hooks folder in `components` package and these hooks can be used in other packages as well.

Reply:
Right but why do we have another one in `shared/src/utils`?

Reply:
I think we should move those to `components` package as well

Reply:
Agree